util/regexp-assemble/data/942280.data

```diff
@@ -3,10 +3,18 @@
 
 ##!+ i
 
+##!> template semicolon ;
```

Comment:
It's obvious that it's a semicolon. Either change the name or add a comment to indicate the semantics, e.g., `single-line-semicolon-comment`.
Is there something special about semicolon? Why isn't `--` an option here?

Reply:
maybe we can keep the semicolon as is without a template? is it better?